src/Jackett.Common/Indexers/NCore.cs

```diff
@@ -20,29 +22,40 @@ public class NCore : BaseWebIndexer
     {
         private string LoginUrl => SiteLink + "login.php";
         private string SearchUrl => SiteLink + "torrents.php";
-        private readonly string[] LanguageCats = { "xvidser", "dvdser", "hdser", "xvid", "dvd", "dvd9", "hd", "mp3", "lossless", "ebook" };
 
-        private new ConfigurationDataNCore configData
+        private new ConfigurationDataNCore configData => (ConfigurationDataNCore)base.configData;
+        private readonly string[] _languageCats =
         {
-            get => (ConfigurationDataNCore)base.configData;
-            set => base.configData = value;
-        }
-
-        public NCore(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
-            : base(name: "nCore",
-                description: "A Hungarian private torrent site.",
-                link: "https://ncore.cc/",
-                caps: new TorznabCapabilities(),
-                configService: configService,
-                client: wc,
-                logger: l,
-                p: ps,
-                configData: new ConfigurationDataNCore())
+            "xvidser",
+            "dvdser",
+            "hdser",
+            "xvid",
+            "dvd",
+            "dvd9",
+            "hd",
+            "mp3",
+            "lossless",
+            "ebook"
+        };
+
+        public NCore(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps) :
+            base("nCore",
+                 description: "A Hungarian private torrent site.",
+                 link: "https://ncore.cc/",
+                 caps: new TorznabCapabilities
+                 {
+                     SupportsImdbMovieSearch = true,
+                     SupportsImdbTVSearch = true
```

Comment:
is this tested with sonarr v3? #8107

Reply:
I haven't tested this. But on a surface level it should. Might need to change a little logic to get it 100% 

The base is there, as the series IMDb ID's are based on the show, not the episodes (test search confirmed), and the multipage processing should allow us to filter out other episodes. But I don't think the code as is filters correctly.

Should I change the logic to filter episodes, or turn off ImdbTV for now?

Reply:
Actually, because of the `query.Offset` logic, filtering out episodes doesn't work, because that will mess with the support of multi-page & limit/offset

It does work, and will return 100 results just fine but won't be able to sort or filter by episode string. Or I can turn it off for now.

Reply:
If this is enabled sonar will try to use it, so if it's not working I prefer it disabled. If it's working let it.

Reply:
It's only partially supported because we can use it and make it work, but it won't always play nice with multipage processing. Disabling for now until we have a better solution.